tasmota/tasmota_xdrv_driver/xdrv_27_esp32_shutter.ino

```diff
@@ -229,6 +229,7 @@ void ShutterSettingsDefault(void) {
     ShutterSettings.shutter_pwmrange[0][i] = pwm_servo_min;
     ShutterSettings.shutter_pwmrange[1][i] = pwm_servo_max;
   }
+  ShutterSettingsSave();
```

Comment:
Are you sure this solves the issue? Doing it here it will overwrite the settings everytime with the default settings resetting the changes the user made.

Reply:
ok agree. let me double check.